side-navigation.yaml

```diff
@@ -106,6 +108,7 @@
       url: /docs/patterns/tabs
     - title: Tooltips
       url: /docs/patterns/tooltips
+  ordering: alphabetical
```

Comment:
Small thing, but I'd suggest moving `ordering` up under `heading`, so it's clearer what it applies to.

So like:

```
- heading: Components
  ordering: alphabetical
  subheadings:
    … 
```

Reply:
Ordering has been moved accordingly